elementary/monitor/fetchers/tests/schema.py

```diff
@@ -25,9 +25,11 @@ class TestResultDBRowSchema(ExtendedBaseModel):
     test_results_description: Optional[str]
     owners: Optional[List[str]]
     model_owner: Optional[List[str]]
-    tags: Optional[List[str]]
+    tags: List[str] = Field(default_factory=list)
```

Comment:
do we still need this one?

Reply:
Well there is other place that uses it and this is the combination of model tags and test_tags.
I guess I will make it a property that return the union of the 2 🙂